src/univers/version_range.py

```diff
@@ -1233,4 +1244,5 @@ def build_range_from_github_advisory_constraint(scheme: str, string: str):
     "nuget": "nuget",
     "pypi": "pypi",
     "packagist": "composer",
+    "conan1": "conan1",
```

Comment:
It should be `"conan" : "conan1"` since gitlab uses conan in their package slug

Reply:
Thank you @TG1999.  I made the change yesterday to `conan1` to address an error I was getting when running this test: 

```
pytest -vvs tests/test_version_range.py::test_conan_gitlab_version_range_parse
```

I just made the change you requested and reran that test, and again I get the same error.  Here's an excerpt from a large terminal output:

```
(venv) Fri Feb 24, 2023 10:00 AM  /home/jmh/dev/nexb/univers jmh (101-univers-support-for-conan)
$ pytest -vvs tests/test_version_range.py::test_conan_gitlab_version_range_parse

[...]

=================================================================================================== FAILURES ====================================================================================================
_______________________________________________________________________________ test_conan_gitlab_version_range_parse[test_case0] _______________________________________________________________________________

test_case = {'expected_vers': 'vers:conan1/>=5.1.0|<=5.1.1', 'native': '>=5.1.0 <=5.1.1', 'scheme': 'conan1', 'test_index': 1}

    @pytest.mark.parametrize("test_case", json.load(open("./tests/data/conan_advisory.json")))
    def test_conan_gitlab_version_range_parse(test_case):
        if test_case["expected_vers"] is None:
            with pytest.raises(InvalidVersion):
                Conan1VersionRange.from_native(string=test_case["native"])
            return
>       result = from_gitlab_native(
            gitlab_scheme=test_case["scheme"],
            string=test_case["native"],
        )

tests/test_version_range.py:364:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

gitlab_scheme = 'conan1', string = '>=5.1.0 <=5.1.1'

    def from_gitlab_native(gitlab_scheme, string):
>       purl_scheme = PURL_TYPE_BY_GITLAB_SCHEME[gitlab_scheme]
E       KeyError: 'conan1'

src/univers/version_range.py:1106: KeyError

[...]

========================================================================================= 427 failed, 1 passed in 8.87s =========================================================================================

(venv) Fri Feb 24, 2023 10:00 AM  /home/jmh/dev/nexb/univers jmh (101-univers-support-for-conan)
$
```

I was unable to identify the source of this error -- do you have any suggestions?

Meanwhile, I will implement your other comments and see if that resolves this set of error messages.